packages/webapp/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:18-slim AS base
```

Comment:
Is the image ever built or do we not even need it? The docks don't have a build image step

Reply:
the image is built by docker compose.

```
  webapp:
    build:
      context: .
```

looks for the Dockerfile in the `.` dir